src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1783,6 +1805,18 @@ private void BindArgumentsAndNames(BracketedArgumentListSyntax argumentListOpt,
                 refKind,
                 allowArglist);
 
+            // check for ref/out property/indexer 
+            if (!hadError && isDelegateCreation && origRefKind != RefKind.None)
```

Comment:
We probably shouldn't enter this ```if``` when result contains more than one argument. For example, more than one argument is given to a delegate creation and this method is called to bind the second argument. Consider including this as a test scenario.